app/models/client.rb

```diff
@@ -32,6 +32,8 @@
 class Client < ApplicationRecord
   devise :lockable, :timeoutable, :trackable
 
+  self.per_page = 25
```

Comment:
This doesn't particularly feel like a Model concern to me, but it seems like this is how will_paginate gem directs us to set these default values, so 👍